src/components/__tests__/menu.js

```diff
@@ -0,0 +1,101 @@
+import Vue from 'vue';
+
+import Menu from '../Menu.vue';
+import { FACET_TREE } from '../../store';
+
+const getFacetValues = jest.fn(attributeName => ({
+  name: attributeName,
+  count: null,
+  isRefined: true,
+  path: null,
+  data: [
+    {
+      count: 319,
+      data: null,
+      isRefined: false,
+      name: 'Bathroom',
+      path: 'Bathroom',
+    },
+    {
+      count: 100,
+      data: null,
+      isRefined: false,
```

Comment:
How about we make one `isRefined = true`? This will allow us to catch some issues regarding class names for example.